src/app/routes/index.tsx

```diff
@@ -182,14 +182,6 @@ const router = createHashRouter([
           </AuthGuard>
         ),
       },
-      // {
-      //   path: 'review-ledger-stx-tx',
-      //   element: <ReviewLedgerStxTransaction />,
-      // },
-      // {
-      //   path: 'review-ledger-ft-tx',
-      //   element: <ReviewLedgerFtTransaction />,
-      // },
```

Comment:
🤔 Are these commented out on purpose?

Reply:
they were relics of pre- stx ledger support, that got removed in another PR which has now been merged to develop

Reply:
😅